modules/engage-paella-player-7/src/plugins/org.opencast.paella.episodesFromSeries.js

```diff
@@ -76,7 +76,7 @@ export default class EpisodesFromSeriesPlugin extends PopUpButtonPlugin {
 
     const thisId = this.player.videoId;
     if (this._episodesData) {
-      const result = this._episodesData['search-results'].result;
+      const result = this._episodesData['result'];
       (Array.isArray(result) ? result : [result]).forEach(({id,dcTitle,mediapackage}) => {
```

Comment:
The dcTitle field is no longer present, moved to dc.title[0], but I can't find where this code is called at all.  If this is dead, can we remove it?

Reply:
If you play a video that is in a series with more videos, the plugin is visible at the top-right, and shows the user a list of the videos in the same series.
Example: https://develop.opencast.org/paella7/ui/watch.html?id=ID-westerberg

<img width="457" alt="Captura de pantalla 2024-02-12 a las 19 44 11" src="https://github.com/opencast/opencast/assets/2735202/0899283a-1349-4393-97cd-cf3335b3a6bf">

Reply:
Ah, thank you @miesgre.  This should now be working, although I just can't seem to get the tests working.  When I test locally, I end up with results from the search endpoints with `localhost:8080` in them, but then the webpack test proxy fails to load the media due to CORS rules.  Any hints on setting up a proper test environment?

Reply:
@gregorydlogan Could you change the `testenv:dev` package.json script to

```
"testenv:dev": "webpack serve --mode development --host=0.0.0.0 --env server=http://localhost:8080",
```

I can't test it now, but I think it should work.
You will need a local opencast server running in 8080 and ingest the `ID-dual-stream-demo` mediapackage.

Reply:
That didn't seem to work, but setting up Opencast to work on :80, then running `npm run dev` and `npx playwright test` after `npm` booted up seemed to work.  Tests pass at least :)